File: Roborio/src/org/usfirst/frc/team6325/robot/subsystems/Climber.java
package org.usfirst.frc.team6325.robot.subsystems;

import org.usfirst.frc.team6325.robot.RobotMap;
import org.usfirst.frc.team6325.robot.commands.ClimberCommand;

import com.ctre.CANTalon;

import edu.wpi.first.wpilibj.command.Subsystem;

/**
 *
 */
public class Climber extends Subsystem {
	
	CANTalon motor = new CANTalon(RobotMap.climber); 
	
	public void drive(double power){
		if(power < 0.1) power = 0;
		motor.set(-power);
	}

	public void initDefaultCommand() {
		// Set the default command for a subsystem here.
		setDefaultCommand(new ClimberCommand());
	}
}

File: Roborio-Kotlin/src/main/java/frc/robot/subsystems/Climber.kt
// Reset Robotics 2019
package frc.robot.subsystems

// Libraries
import org.sertain.command.Subsystem
import org.sertain.RobotLifecycle
import edu.wpi.cscore.UsbCamera
import com.ctre.phoenix.motorcontrol.*
import com.ctre.phoenix.motorcontrol.can.*
import edu.wpi.first.wpilibj.CameraServer
import frc.robot.data.ClimberData
import frc.robot.commands.Climber.ClimberJoystickDrive


public object Climber : Subsystem() 
{
    val climberData: ClimberData = ClimberData()

    val motor: WPI_TalonSRX = WPI_TalonSRX(climberData.motor)
	
	fun drive(power: Double)
    {
		motor.set(power)
	}

    override val defaultCommand = ClimberJoystickDrive()
}